src/pages/workspaces/workspace/analysis/utils/tool-utils.ts

```diff
@@ -179,3 +180,16 @@ export const isToolHidden = (toolLabel: ToolLabel, cloudProvider: CloudProvider)
   [toolLabel === appToolLabels.CROMWELL && cloudProvider === cloudProviderTypes.GCP && !isCromwellAppVisible(), () => true],
   [Utils.DEFAULT, () => false]
 )
+
+export const getCurrentMountDirectory = (toolLabel: ToolLabel) => {
```

Comment:
You define `getCurrentMountDirectory` twice in this PR in different places with different signatures